python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -946,6 +946,291 @@ router bgp {{ router_bgp.as }}
+      neighbor {{ neighbor.ip_address }} aigp-session
+{%             endif %}
+{%             if neighbor.multi_path is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} multi-path
+{%             endif %}
+{%             if neighbor.next_hop_resolution.v4_mapped_v6_translation is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} next-hop resolution v4-mapped-v6 translation
+{%             elif neighbor.next_hop_resolution.v4_mapped_v6_translation is arista.avd.defined(false) %}
+      neighbor {{ neighbor.ip_address }} next-hop resolution v4-mapped-v6 translation disabled
+{%             endif %}
+{%         endfor %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.networks is arista.avd.defined %}
+{%             for network in router_bgp.address_family_ipv4_labeled_unicast.networks %}
+{%                 set network_cli = "network " ~ network.prefix %}
+{%                 if network.route_map is arista.avd.defined %}
+{%                     set network_cli = network_cli ~ " route-map " ~ network.route_map %}
+{%                 endif %}
+      {{ network_cli }}
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.next_hops is arista.avd.defined %}
+{%             for next_hop in router_bgp.address_family_ipv4_labeled_unicast.next_hops %}
+{%                 set next_hop_cli = "next-hop " ~ next_hop.ip_address ~ " originate" %}
+{%                 if next_hop.lfib_backup_ip_forwarding is arista.avd.defined(true) %}
+{%                     set next_hop_cli = next_hop_cli ~ " lfib-backup ip-forwarding" %}
+{%                 endif %}
+      {{ next_hop_cli }}
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.lfib_entry_installation_skipped is arista.avd.defined(true) %}
+      lfib entry installation skipped
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.label_local_termination is arista.avd.defined %}
+      label local-termination {{ router_bgp.address_family_ipv4_labeled_unicast.label_local_termination }}
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.graceful_restart is arista.avd.defined(true) %}
+      graceful-restart
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.neighbor_default.next_hop_self is arista.avd.defined(true) %}
+      neighbor default next-hop-self
+{%         endif %}
+{%         if router_bgp.address_family_ipv4_labeled_unicast.next_hop_resolution_ribs is arista.avd.defined %}
+{%             set v4_bgp_lu_resolution_ribs = [] %}
+{%             for rib in router_bgp.address_family_ipv4_labeled_unicast.next_hop_resolution_ribs %}
+{%                 if rib.rib_type is arista.avd.defined("tunnel-rib-colored") %}
+{%                     do v4_bgp_lu_resolution_ribs.append("tunnel-rib-colored system-colored-tunnel-rib") %}
```

Comment:
```suggestion
{%                     do v4_bgp_lu_resolution_ribs.append("tunnel-rib colored system-colored-tunnel-rib") %}
```

Reply:
EOS also supports below command:
      next-hop resolution ribs system-connected tunnel-rib colored system-colored-tunnel-rib

Reply:
If this is wrong here, it is also wrong in other address families. Please confirm again.

Reply:
For evpn we have `next-hop mpls resolution ribs tunnel-rib-colored system-colored-tunnel-rib tunnel-rib test-rib system-connected` so notice that both `tunnel-rib` and `tunnel-rib-colored` are valid values.

Reply:
But it is not working with ipv4 LU
```
s1-leaf1(config-router-bgp-af-label)#next-hop resolution ribs tunnel-rib-colored ?
% Unrecognized command
```

Reply:
On 4.32.2F, it is not working for evpn also.

Reply:
Checked on some older EOS versions as well, and indeed it is wrong in EVPN as well. It should be `tunnel-rib colored` instead of `tunnel-rib-colored`.

Reply:
Fixed.